### PR TITLE
Defensive coding + deleted obsolete files

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* None

*Description of changes:*
- updated app version (24)
- deleted obsolete Invoice component
- added defensive code (don't load NR data if get failed)
- added spinner to Existing Name Request refresh request
- deleted unused Api Error Mixin
- deleted obsolete Name Request Mixin
- added return code to fetchStats (for future use)
- deleted obsolete New Request Module
- added spinner to Landing page when loading existing NR (ie, NR id in URL)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Comment by kialj876** [comment]
Hey sev, can you keep me in the loop for when this is merged? My incoming changes will have a few conflicts (nothing too major)

**Review by kialj876 — APPROVED** [review]

File: src/components/existing-request/existing-request-display.vue [hunk 1]
```diff
@@ -249,9 +249,6 @@ export default class ExistingRequestDisplay extends Mixins(
     return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
   }
 
-  /** This is True while refreshing the NR. (Not used in template at the moment.) */
-  checking = false
```
Comment (cameron-freshworks) [inline src/components/existing-request/existing-request-display.vue:253]: any idea what this was used for previously 😱

**Review by cameron-freshworks — APPROVED** [review]
Looks great!

Merge whenever you are ready 👍

File: src/components/existing-request/existing-request-display.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/existing-request/existing-request-display.vue:253]: There are a couple of state variables that are used in the UI. I assume this was one of them that became obsolete either before we started working on this page or in the early days when we refactored this page per UI designs.
I'm not worried about this.

**Pull request merged**